src/posts/PostList.tsx

```diff
@@ -1,43 +1,47 @@
 import { useState, useEffect } from 'react';
 import axios from 'axios';
 import Post from './Post';
-import { useFilterStore } from '../helpers/filter-store';
+import { PostsStore } from '../helpers/post-store';
 import "./PostList.scss"
+import Spinner from './Spinner';
+
+const API_LINK = 'https://mocki.io/v1/418eafe2-1002-4145-94f2-370a4eb34be8';
+interface Post {
+    is_verified: boolean;
+}
 
 const PostList = () => {
-  const { verified } = useFilterStore();
-  const [posts, setPosts] = useState([] as Array<{ is_verified: boolean }>);
+  const { verified } = PostsStore();
+  const [posts, setPosts] = useState<Post[]>([]);
```

Comment:
store the original posts inside `posts-store` so it can be accessed by any inherited component